packages/chain-bridge/src/adapters/StargateAdapter.sol

```diff
@@ -400,16 +466,20 @@ contract StargateAdapter is
                     gasLimit: options.gasLimit,
```

Comment:
Why are we recreating BridgeOptions here when the parameter `options` is already of the right type and we are not overriding any values?